### ISSUE TITLE
Type error with address field

### ISSUE THREAD

**Issue by pixelmachine** [barrelstrength/craft-sprout-forms#609]
I've added an address field to my form and I'm getting an error regardless of any other configuration.

```php
CommerceGuys\Addressing\Subdivision\SubdivisionRepository::createSubdivisionFromDefinitions(): 
Argument #1 ($code) must be of type string, null given, called in 
/craft3/vendor/commerceguys/addressing/src/Subdivision/SubdivisionRepository.php on line 62
```
Latest Craft Pro and Commerce

**Comment by pixelmachine** [comment]
Update: I've tried reinstalling, disabling all other plugins, trialing Pro etc. etc. and nothing seems to work.

**Comment by KBjana** [comment]
I've just updated from Craft 3.7.16 to Craft 3.7.55.3 and am now having the same issue. I also updated Sprout Forms from 3.13.7 to 3.13.8 but downgrading doesn't fix the issue so I believe it's a Craft update that's caused the error

**Comment by KasimMahroof** [comment]
I've had the same issue and managed to track down the cause to the [barrelstrength/sprout-base-fields](https://github.com/barrelstrength/craft-sprout-base-fields) dependency that is specified in this project's composer.json.

In the [barrelstrength/sprout-base-fields](https://github.com/barrelstrength/craft-sprout-base-fields) project, it has an issue with a dependency which I've made a pull request for: https://github.com/barrelstrength/craft-sprout-base-fields/pull/33

As a temp fix, in your Craft CMS project's composer.json, add the following:

```json
"require": {
...
"commerceguys/addressing": "1.3.0"
}
```

Then run:
```bash
composer update
```

This should be a good temp fix before the other package get's fixed.

**Comment by BenParizek** [comment]
I've just released v3.13.9 which should resolve this issue.
